spec/controllers/verify/confirmations_controller_spec.rb

```diff
@@ -86,6 +86,13 @@ def stub_idv_session
 
         get :index
       end
+
+      it 'creates an `account_verified` event once per confirmation' do
+        get :index
+        user.reload
+
+        expect(user.events.where(event_type: 8).size).to be 1
```

Comment:
enums give us a handy shorthand so you can call `user.events.account_verified` and it will return all of the events for that enum type. Seems like that might be clearer than looking for the integer `8`